subprojects/core/src/test/groovy/org/gradle/execution/DryRunBuildExecutionActionTest.groovy

```diff
@@ -55,7 +54,8 @@ class DryRunBuildExecutionActionTest extends Specification {
         action.execute(gradle, executionPlan)
 
         then:
-        textOutputFactory.toString() == "{$category}:task1 {progressstatus}SKIPPED${EOL}{$category}:task2 {progressstatus}SKIPPED$EOL"
+        textOutputFactory.category == DryRunBuildExecutionAction.canonicalName
+        textOutputFactory.output == ":task1 {progressstatus}SKIPPED${EOL}:task2 {progressstatus}SKIPPED$EOL"
```

Comment:
💅 EOL is `SystemProperties.instance.lineSeparator` in this test instead of `TextUtil.platformLineSeparator`?  They're the same thing, of course, but we have a fixture that allows us to have a single way to determine this.  I feel like we should use it everywhere or nowhere...